### PR TITLE
Enable opts as second arg in build, buildTree & buildSFX

### PR DESCRIPTION
**Pull request by crisptrutski**
This is just #142 with a bugfix

### CODE REVIEW

File: lib/builder.js [hunk 1]
```diff
@@ -30,6 +30,13 @@ function processOpts(opts_, outFile) {
   return opts;
 }
 
+function isPlainObject(obj) {
```
Comment (guybedford) [inline lib/builder.js]: Not sure I understand the value of all this still.

File: lib/builder.js [hunk 2]
```diff
@@ -141,6 +156,14 @@ function addExtraOutputs(output, tree, opts) {
 Builder.prototype.buildTree = function(tree, outFile, opts) {
   var loader = this.loader;
   var self = this;
+
+  // allow passing opts
+  // as second argument.
+  if(isPlainObject(outFile)){
```
Comment (guybedford) [inline lib/builder.js]: if (typeof outFile === 'object' && outFile !== null)

Comment (crisptrutski) [inline lib/builder.js]: Even better `if (outFile && typeof outFile === 'object')` IMHO (less characters and faster). Yeah I'm happy with that, don't see why we care about prototype. An "actual" `outFile` parameter certainly wouldn't be a class instance.

**Comment by crisptrutski** [comment]
Oh man, broke it again. Thumbelina

**Comment by guybedford** [comment]
Beautiful. Squash and merge?

**Comment by crisptrutski** [comment]
Just wanted to remove the my commit pun? :sob:

**Pull request merged**